### ISSUE TITLE
Loss type for the action branch

### ISSUE THREAD

**Issue by kennymckormick** [holistic-video-understanding/HVU-Dataset#6]
Thanks for releasing the HVU Dataset. In your paper, you say that you use 'softmax loss for the action recognition branch'. However, there are multiple action labels for some videos in the HVU dataset. So I wonder how u use softmax loss for action recognition.

**Comment by alidiba67** [comment]
Hi, For each head, we used the binary cross-entropy as you can also see in the final version of work at ECCV:
https://pages.iai.uni-bonn.de/gall_juergen/download/HVU_eccv20.pdf
